ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```
@@ -15,12 +33,16 @@ tests:
 - as: unit-tests
   commands: ci/prow/unit-tests
   container:
-    from: src
+    clone: true
+    from: ocp_builder_rhel-8-golang-1.18-openshift-4.12
 - as: lint
-  commands: ci/prow/lint
+  commands: |
+    export GOCACHE=/tmp
+    export GOLANGCI_LINT_CACHE=/tmp
+    make lint GOLANGCI_LINT=/usr/bin/golangci-lint
```

```suggestion
    ci/prow/lint
```
Commented on the repo PR as well.